src/birthder.js

```diff
@@ -258,14 +259,14 @@
             return msg.send(message);
         });
 
-        // Regularly checks for a birthday, announces to "generic" chat room
+        // Check regularly if today is someone's birthday, write birthday messages to the general channel.
         if (BIRTHDAY_CRON_STRING) {
             schedule.scheduleJob(BIRTHDAY_CRON_STRING, () => sendCongratulations(robot));
         }
 
-        // Announce birthdays to each user (except one whose birthday it is) in advance
+        // Send reminders to the users of the upcoming birthdays (except ones whose birthday it is).
```

Comment:
"Send upcoming birthdays reminders" - ?
or 
"Send reminders of the upcoming birthdays to the users" - ?

Reply:
I like the second one and will use it.